scripts/image.py

```diff
@@ -0,0 +1,426 @@
+import os
+import numpy as np
+import nibabel as nib
+import logging
+from copy import deepcopy
+
+logger = logging.getLogger(__name__)
+
+class Image(object):
+    """
+    Compacted version of the https://github.com/spinalcordtoolbox/spinalcordtoolbox/image.py image module
+    Create an object that behaves similarly to nibabel's image object. Useful additions include: dims, change_orientation and getNonZeroCoordinates.
```

Comment:
Maybe you could also bring `zeros_like`, `empty_like`, `find_zmin_zmax‎` functions into this light-weight class. Those are also useful